build.sh

```diff
@@ -179,3 +188,13 @@ if (( ${NUMARGS} == 0 )) || hasArg cuspatial; then
         python setup.py install --single-version-externally-managed --record=record.txt -- -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX} -DCMAKE_LIBRARY_PATH=${LIBCUSPATIAL_BUILD_DIR} ${EXTRA_CMAKE_ARGS}
     fi
 fi
+
+# Build and install the cuproj Python package
```

Comment:
Looks like the code for building libcuproj is missing?

Reply:
libcuproj is header-only. There's no binary to build. There are libcuproj tests though.

Reply:
Yep, it is built when you build libcuspatial. They are not meant to be independent yet. But since I don't know how to make a wheel with both in it, the wheels are separate.

Reply:
It sounds like libcuproj is header-only and its CMake configuration happens as part of libcuspatial, but it does install a separate set of headers and provides its own CMake config modules etc as a separate project, is that right (seems roughly correct from a skim of all the code)? So then the Python cuproj can find just libcuproj and not libcuspatial because it truly is a separate C++ package?

Reply:
Correct. Python cuproj does not need libcuspatial.